src/index.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
```

Comment:
this won't break anything, but since you never use Route or Link within index.js it's unnecessary to import them along with BrowserRouter